internal/configuration.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import "github.com/caarlos0/env/v11"
+
+type Configuration struct {
+	GithubToken string `env:"GITHUB_TOKEN,notEmpty"`
+	OpenAIToken string `env:"OPENAI_API_KEY,notEmpty"`
+}
```

Comment:
Consider returning more structured error messages. Instead of returning the default parsing error, you could wrap it using `fmt.Errorf` to provide context about which configuration failed to load, increasing debuggability.